FreeRTOS-Plus/Demo/FreeRTOS_Cellular_Interface_Windows_Simulator/Common/comm_if_windows.c

```diff
@@ -83,7 +82,7 @@
 
 /*-----------------------------------------------------------*/
 
-typedef struct _cellularCommContext
+typedef struct cellularCommContext
```

Comment:
cellularCommContext --> CellularCommContext

Reply:
cellularCommContext is comm interface windows internal structure. So small letter is used here.